client/src/components/ui/itemrow/util.ts

```diff
@@ -1,5 +1,10 @@
+import { DeferredSnapshot } from '~model/api';
 import { Document } from '~model/document';
 
 export function goToTrackingPage(id: Document['id']) {
     window.location.href = `/track?id=${id}`;
+}
+
+export function markDeferred(store: DeferredSnapshot[], doc: Document['id']) {
```

Comment:
I disagree with the naming. I believe `findDeferredSnapshot` expresses the intent much clearer.
```suggestion
export function findDeferredSnapshot(store: DeferredSnapshot[], doc: Document['id']) {
```